documentation/deployment.md

```diff
@@ -46,3 +51,8 @@ Do you want to perform these actions in workspace "testing"?
 
 #### Step 3
 Once the deployment has finished, review the output of the script to ensure everything succeeded by checking AWS to make sure the expected changes were made.
+
+## Upgrading Terraform
+We have locked Terraform to its major version to protect the team from accidentally running a deployment from a version not compatible with the Terraform files we have. This will hopefully give us explicit feedback on version issues **before** being presented with a barrage of Terraform errors throughout all the files during the last phase of a production deployment.
```

Comment:
Terraform uses the pre-release form of semver, where minor version changes are actually major releases. So if we're pinning, we should pin to minor version. See e.g. 0.12's draft upgrade guide: https://www.terraform.io/upgrade-guides/0-12.html

Reply:
Good spot 

> Terraform v0.12 will be a major release focused on configuration language improvements and thus will include some changes that you'll need to consider when upgrading.